### PR TITLE
(RNA) Sequencing Update for Portal Containers

### PR DESCRIPTION
**Pull request by ilan-gold**
Forgot that @mruffalo was making the `hubmap_ui` directory so need to merge https://github.com/hubmapconsortium/portal-containers/pull/61 first and re-update.

### CODE REVIEW

**Comment by ilan-gold** [comment]
Ready to go @jswelling! Hope this works!

File: src/ingest-pipeline/airflow/dags/salmon_rnaseq.py [hunk 1]
```diff
@@ -167,12 +208,23 @@ def build_cwltool_cmd2(**kwargs):
...
         )
 
+        t_maybe_keep_cwl3 = BranchPythonOperator(
+            task_id="maybe_keep_cwl3",
+            python_callable=utils.pythonop_maybe_keep,
+            provide_context=True,
+            op_kwargs={
+                "next_op": "move_data",
+                "bail_op": "set_dataset_error",
+                "test_op": "t_convert_for_ui_2",
```
Comment (jswelling) [inline src/ingest-pipeline/airflow/dags/salmon_rnaseq.py:224]: test_op should be "convert_for_ui_2", the task_id of the step.  t_convert_for_ui_2 is the name of the python variable pointing to the step.

**Review by jswelling — CHANGES_REQUESTED** [review]
I found one typo, at line 224 of salmon_rnaseq.py- see below.  Also, it looks like the update undoes a bunch of deletions of obsolete CWL pipelines that @mruffalo has been working throught.  Is that intended?

**Review by jswelling — APPROVED** [review]
Thanks!

**Pull request merged**